services/wallet/bridge/cbridge.go

```diff
@@ -377,3 +377,11 @@ func (s *CBridge) CalculateAmountOut(from, to *params.Network, amountIn *big.Int
 	amountOut, _ := new(big.Int).SetString(amt.EqValueTokenAmt, 10)
 	return amountOut, nil
 }
+
+func (s *CBridge) ValidateTransaction(sendArgs *TransactionBridge) (bool, error) {
+	if sendArgs.CbridgeTx == nil {
```

Comment:
seems like the returned boolean is just (err == nil), do we even need it?

Reply:
what do you mean err == nil, i am also returning a message, is it what you want me to remove?